apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -74,6 +74,23 @@ export default class TeacherHomepage extends Component {
           />
         )}
 
+        {hocLaunch &&
+         hocLaunch.special_announcement &&
+         hocLaunch.special_announcement === "celebs2017actualhoc" &&
+         !isEnglish && (
+          <SpecialAnnouncementActionBlock
+            isRtl={isRtl}
+            imageUrl={pegasus('/images/mc/fill-540x289/special-announcement-hoc2017.jpg')}
+            heading={i18n.specialAnnouncementHeadingCelebs()}
```

Comment:
Just want to call out that this becomes a footgun if we try to change this heading later on.